clouds/cli/README.md

```diff
@@ -0,0 +1,51 @@
+# Cloud nuke CLI tools
+* To run with `cloud-cli`:
+```bash
+pipx install redhat-qe-cloud-tools
+cloud-cli --help
+```
+
+
+### AWS nuke
+#### Pre-requisites:
+- Install `cloud-nuke` CLI tool (see https://github.com/gruntwork-io/cloud-nuke)
+
+#### Run nuke for specific AWS regions cloud resources:
+* Using Poetry:
+```bash
```

Comment:
missing new line before this line (in all places - a new line should come before )

Reply:
done

Reply:
done